docs/docker-hub/builds.md

```diff
@@ -398,9 +398,8 @@ Webhooks are available under the Settings menu of each Repository.
 > **Note:** If you want to test your webhook out we recommend using
 > a tool like [requestb.in](http://requestb.in/).
 
-> **Note**: The Docker Hub servers are currently in the IP range
-> `162.242.195.64 - 162.242.195.127`, so you can restrict your webhooks to
-> accept webhook requests from that set of IP addresses.
+> **Note**: The Docker Hub servers use an elastic IP range, so can't be
```

Comment:
Two notes in a row look a bit odd. Rather than form a note list, you can join the top note with the previous sentence. Also, the second note is a bit hard to parse:

-->

Webhooks are available under the "Settings" menu of each Repository. If you want to test your webhook, you can use a tool like [requestb.in](http://requestb.in/).

>**Note**: The Docker Hub servers use an elastic IP range, so you can't filter for them by IP address.